app/templates/briefs/clarification_question.html

```diff
@@ -63,16 +63,15 @@
         {% include 'toolkit/page-heading.html' %}
       {% endwith %}
 
-
-      <div class="hint">
-        <p>Your question will be published with the buyers’ response by {{ brief.clarificationQuestionsPublishedBy | dateformat }}.</p>
-        <p>All questions and answers will be posted on the Digital Marketplace. Your company name won’t be visible.</p>
-        <p>You shouldn’t include any confidential information in your question.</p>
-        <p>Read more about <a href="https://www.gov.uk/guidance/how-to-answer-supplier-questions-about-your-digital-outcomes-and-specialists-requirements">how supplier questions are managed</a>.</p>
-      </div>
-
       <form method="post">
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+        {% set question_advice =
```

Comment:
I don't get how these `question_advice` things get rendered?  They are set as a variable but then never passed in to anything?  What is this black magic?